declarations/Speechify.json

```diff
@@ -11,8 +11,8 @@
       "fetch": "https://speechify.com/privacy/",
       "select": [
         {
-          "startBefore": "#cookiepolicy",
-          "endBefore": "[id=\"6\"]"
+          "startBefore": "#cookie-policy",
+          "endBefore": "#contact-us"
```

Comment:
This contact us section is a part of the Trackers Policy
```suggestion
          "endAfter": "#contact-us"
```

Reply:
You're right, but your suggestion doesn't allow to select the paragraph after the title "Contact Us".

Reply:
What do you think of
```suggestion
          "endBefore": ".footertopwave"
```

Reply:
> You're right, but your suggestion doesn't allow to select the paragraph after the title "Contact Us".

Yes, sorry, I made a proposal too quickly.

Reply:
It seems to me that this is more robust:
```suggestion
          "endAfter": "main"
```

Reply:
> Yes, sorry, I made a proposal too quickly.

Don't worry, and thank you for pointing out this error ;)